### PR TITLE
46418108_YOLO_ISIC Pull Request

### PR DESCRIPTION
**Pull request by nikmatha**
Use of YOLOv5 to train a model on ISIC 2017 dataset to detect lesions in different images and determine whether it is benign or melanoma.

### CODE REVIEW

**Comment by nathasha-naranpanawa** [comment]
- Has committed all train/validation/test set images and label files! Need to be removed!
- Commit messages are good.
- Readme is good, with plots provided for all metrics.
- No classification accuracy measured.
- Performance is good. Has reached 0.8 IoU for both small and medium models
- Seems to have run the default settings of ultralytics/YOLOv5 repo and no code alterations or hyperparameter tuning done by themselves, so almost all of the work done is only for preprocessing the data (which also seems to have been done with a 3rd party software).

**Comment by shakes76** [comment]
## Good Practice (Design/Commenting, TF/Torch Usage)
Adequate use and implementation, no hyper-parameter tuning done/tried -2
Good spacing and comments
Header blocks missing -1
## Recognition Problem 
Solves problem (no classification accuracy) -2
Driver Script present
File structure present
Shows Usage & Demo & Visualisation & Data usage 
Module present
Commenting
No Data leakage
Difficulty: Normal -5
## Commit Log
Meaningful commit messages
Progressive commits used
## Documentation
ReadMe acceptable/good
Good Description and Comments 
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
Feedback was required, done, but won't be merging because data is in history now, but you have received your feedback mark -2 +2
Request Description OK could be more informative -1
